fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1814,6 +1814,70 @@ public void replayRecoverPartition(RecoverInfo info) {
         localMetastore.replayRecoverPartition(info);
     }
 
+    public static String getMaterializedViewDdtStmt(MaterializedView mv) {
```

Comment:
what Ddt mean ? do you mean getMaterializedViewDdlStmt?

Reply:
typo, and should be `Ddl`